Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Stage 1
+FROM mcr.microsoft.com/dotnet/sdk:6.0 as build
+WORKDIR /build
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY ./Dfe.Academies.Academisation.Core/ ./Dfe.Academies.Academisation.Core/
+COPY ./Dfe.Academies.Academisation.Data/ ./Dfe.Academies.Academisation.Data/
+COPY ./Dfe.Academies.Academisation.Domain/ ./Dfe.Academies.Academisation.Domain/
+COPY ./Dfe.Academies.Academisation.Domain.Core/ ./Dfe.Academies.Academisation.Domain.Core/
+COPY ./Dfe.Academies.Academisation.IData/ ./Dfe.Academies.Academisation.IData/
+COPY ./Dfe.Academies.Academisation.IDomain/ ./Dfe.Academies.Academisation.IDomain/
+COPY ./Dfe.Academies.Academisation.IService/ ./Dfe.Academies.Academisation.IService/
+COPY ./Dfe.Academies.Academisation.Service/ ./Dfe.Academies.Academisation.Service/
+COPY ./Dfe.Academies.Academisation.WebApi/ ./Dfe.Academies.Academisation.WebApi/
+COPY ./Dfe.Academies.Academisation.sln/ ./Dfe.Academies.Academisation.sln
+
+RUN dotnet restore Dfe.Academies.Academisation.WebApi
+RUN dotnet build Dfe.Academies.Academisation.WebApi
+
+RUN dotnet new tool-manifest
+RUN dotnet tool install dotnet-ef
+
+RUN dotnet ef migrations script --output /app/SQL/DbMigrationScript.sql --idempotent -p /build/Dfe.Academies.Academisation.Data -s /build/Dfe.Academies.Academisation.WebApi -c AcademisationContext
```

Comment:
do we need to spit out the migration SQL file?

Reply:
This is used in the docker-entrypoint, imported into the database